### PR TITLE
Feature/version 5

### PR DESCRIPTION
**Pull request by DASPRiD**

### CODE REVIEW

**Comment by jeremiahsmall** [comment]
Still needs unit tests, obviously, and also has a couple phpcs sniffs.

File: composer.json [hunk 1]
```diff
@@ -14,12 +14,10 @@
     "require": {
         "php": "^5.5 || ^7.0",
```
Comment (jeremiahsmall) [inline composer.json]: Remove 5.5 support, and add "ext-bcmath" requirement.

File: src/Client/Exception/FileMakerException.php [hunk 2]
```diff
@@ -183,14 +198,12 @@
...
         922 => 'User dictionary cannot be found',
         923 => 'User dictionary is read-only',
-        951 => 'An unexpected error occurred (CWP)',
-        954 => 'Unsupported XML grammar (CWP)',
-        955 => 'No database name (CWP)',
-        956 => 'Maximum number of database sessions exceeded (CWP)',
-        957 => 'Conflicting commands (CWP)',
-        958 => 'Parameter missing (CWP)',
-        959 => 'Custom Web Publishing technology disabled (CWP)',
-        960 => 'Parameter is invalid (CWP)',
```
Comment (jeremiahsmall) [inline src/Client/Exception/FileMakerException.php]: Where did you source the updated errors from? Did they actually remove a couple of error codes?

Comment (DASPRiD) [inline src/Client/Exception/FileMakerException.php]: I used their latest online document.

**Comment by jeremiahsmall** [comment]
Squashed commits and remote merged with ff47056ef4d624390d356593b14f26a17fa73fb4

**Pull request closed without merging**